processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorSubscription.java

```diff
@@ -147,6 +146,7 @@ public void receive(ConsumerRecord<byte[], byte[]> record) {
         if (props.get(CONFIG_BIND_CLIENT_METRICS).value()) {
             metrics.bindClientMetrics(consumer);
         }
+        quotaApplier = quotaApplierSupplier.get();
```

Comment:
I wanted to unify any resource (that may need clean-up at ProcessorSubscription close) initialization to inside the ProcessorSubscription. (like ConsumerSupplier)

Reply:
Discussed verbally.

Though it's not a strict rule, using constructor only for field initialization and doing resource instantiation outside of it would be more practical.
I'll fix like that